.github/CODEOWNERS

```diff
@@ -175,8 +175,7 @@
 
 
 # Dev workflow team.
-# TODO(#13596): Add @krishnarao22 back to codeowners after 2021-09-01
-/.circleci/ @SAEb-ai @U8NWXD
+/.circleci/ @oppia/automated-qa-reviewers
```

Comment:
Can we close #13596 once this PR is merged? (If yes then let's add "fixes #13596" in the PR description)

Reply:
done